### PR TITLE
fix Decisions matrix download

### PR DESCRIPTION
**Pull request by sythel**
fix downloadHandler for Decisions

### CODE REVIEW

File: src/dams_mcda/server.R [hunk 1]
```diff
@@ -1477,17 +1477,23 @@ server <- function(input, output, session) {
...
+
 		# download for Decision Matrix
 		output[[paste0("DownloadDecisionMatrix", damId)]] <- downloadHandler(
-		  filename = "DecisionMatrices_All.csv"	)
-		
+		  filename = function() {
+			  format(Sys.time(), "DecisionMatrices_All_%Y-%m-%d_%H-%M-%S_%z.csv")
+		  },
+		  content = function(file) {
+			  write.csv( Decisions, file, row.names = TRUE, quote=TRUE)
```
Comment (sythel) [inline src/dams_mcda/server.R]: @elbfox we might not need rownames, I will launch this branch to shiny2 for you to decide

Comment (sythel) [reply src/dams_mcda/server.R]: doesn't seem to be functional, investigating

Comment (elbfox) [reply src/dams_mcda/server.R]: I did the extension as .csv instead of .xlsx...I'll go fix it in #188. Sorry!

Comment (sythel) [reply src/dams_mcda/server.R]: @elbfox There is no Decisions object anymore. Does it need to be re-added or is there a different data object you are trying to serve as a download?

Comment (sythel) [reply src/dams_mcda/server.R]: oh its a existing file; will update this branch

File: src/dams_mcda/server.R [hunk 2]
```diff
@@ -1477,17 +1477,23 @@ server <- function(input, output, session) {
...
 		  }
 		)
-		
+
 		# download for Decision Matrix
 		output[[paste0("DownloadDecisionMatrix", damId)]] <- downloadHandler(
-		  filename = "DecisionMatrices_All.csv"	)
-		
+		  filename = function() {
+			  format(Sys.time(), "DecisionMatrices_All_%Y-%m-%d_%H-%M-%S_%z.csv")
```
[the remark this answers is not in the document]
Comment (sythel) [reply src/dams_mcda/server.R]: if we keep a date & time with each download it can help debugging for future problems. We can remove it.

**Pull request merged**